doobie-hikari/src/main/scala/com/avast/sst/doobie/DoobieHikariConfig.scala

```diff
@@ -0,0 +1,18 @@
+package com.avast.sst.doobie
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.FiniteDuration
+
+final case class DoobieHikariConfig(driver: String,
```

Comment:
and to be super duper faithful to the original config, shouldn't we add the missing fields?

https://github.com/openbouquet/HikariCP/blob/master/src/main/java/com/zaxxer/hikari/HikariConfig.java#L58

Reply:
Done.